### PR TITLE
finished model

### PR DESCRIPTION
**Pull request by ninhunest**

### CODE REVIEW

**Comment by khactoan** [comment]
approved

**Comment by ninhunest** [comment]
ready

File: db/migrate/20171207060556_create_books.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
+      t.string :title, null: false
+      t.text :description, null: false
+      t.string :image_url
+      t.decimal :price, :precision => 8, null: false
```
Comment (huongnguyenmta) [inline db/migrate/20171207060556_create_books.rb]: `precision: 8 `

File: db/migrate/20171207060831_create_rates.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateRates < ActiveRecord::Migration[5.1]
+  def change
+    create_table :rates do |t|
+      t.integer :rate,default: 0
```
Comment (huongnguyenmta) [inline db/migrate/20171207060831_create_rates.rb]: thiếu khoảng trắng

**Comment by huongnguyenmta** [comment]
commented

**Comment by ninhunest** [comment]
ready

**Pull request merged**